ouroboros-network-protocols/test-cddl/Main.hs

```diff
@@ -429,6 +429,7 @@ localStateQueryCodec :: Codec (LocalStateQuery Block BlockPoint Query)
                               CBOR.DeserialiseFailure IO BL.ByteString
 localStateQueryCodec =
     codecLocalStateQuery
+      maxBound
```

Comment:
It's better if we use a random value.  This is not useful for the changes in this PR, but rather for possible future changes.

Reply:
I don't know the architecture of that `test-cddl` code (would the properties also depend on the chosen version?), so I'd like to leave it as `maxBound` for now, kicking the can down the road to whoever introduces protocol versions for which this would be useful.

Reply:
I created an issue for it: https://github.com/IntersectMBO/ouroboros-network/issues/4772